client/src/transmitter/index.html

```diff
@@ -7,32 +7,20 @@
 
     <title>Just Cast It</title>
 
-    <base href="/transmitter/">
-
-    <link href="https://vjs.zencdn.net/7.11.4/video-js.css" rel="stylesheet" />
-    <link
-      href="https://unpkg.com/@silvermine/videojs-quality-selector/dist/css/quality-selector.css"
-      rel="stylesheet"
-    />
+    <base href="/transmitter/" />
 
     <link href="index.css" rel="stylesheet" />
   </head>
   <body>
-    <video
-      id="video"
-      class="video-js vjs-default-skin"
-      autoplay
-      data-setup='{ "controls": false, "autoplay": true, "preload": "auto" }'
-    ></video>
+    <video id="video" autoplay></video>
```

Comment:
Do we even need this `video` element too? Why is it not only the canvas with some video being drawn inside it?

Reply:
`video` must be "rendered" for `canvas.drawImage` to work.